pkg/pool-weighted/contracts/BaseWeightedPool.sol

```diff
@@ -455,6 +478,10 @@ abstract contract BaseWeightedPool is BaseMinimalSwapInfoPool, WeightedMath {
         );
         _require(bptAmountIn <= maxBPTAmountIn, Errors.BPT_IN_MAX_AMOUNT);
 
+        // This in exceptional situation in which the fees are charged on tokens out instead of a token in. Note that
+        // swapFees is already upscaled.
```

Comment:
```suggestion
        // This is an exceptional situation in which the fee is charged on a token out instead of a token in.
        // Note that swapFee is already upscaled.
```